### PR TITLE
ALFREDAPI-544 fix Date range search

### PR DESCRIPTION
**Pull request by hechmi-dammak-xenit**
Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-<**YOUR TICKET ID**>

- [ ] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [ ] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [user guide](https://docs.xenit.eu/alfred-api/user/rest-api/index.html#rest-http-result-codes)?
- [ ] Is error handling done through a method annotated with `@ExceptionHandler` in the webscript classes?
- [ ] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [ ] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

**Comment by WimCrols** [comment]
Please extend CHANGELOG.md.

**Comment by hechmi-dammak-xenit** [comment]
> Please extend CHANGELOG.md.

done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Alfred API - Changelog
 
+## 5.0.2 (2024-05-13)
+
+The artifact name of `apix-interface` has been changed to `alfred-api-interface`.
```
Comment (WimCrols) [inline CHANGELOG.md:5]: You can remove this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Alfred API - Changelog
 
+## 5.0.2 (2024-05-13)
```
Comment (WimCrols) [inline CHANGELOG.md:3]: Please change date to `(unreleased)`

(and I see now that the post-release commit was forgotten in the previous release. Thanks for adding it.)

**Review by WimCrols — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (todorinskiz) [reply CHANGELOG.md:3]: We need this release to push Vespa to production on ScaleWay

Comment (WimCrols) [reply CHANGELOG.md:3]: I'm not following... Sure, you can release it. This is always a separate ticket that fill set the date here.

**Review by todorinskiz — APPROVED** [review]

**Review by WimCrols — APPROVED** [review]

**Pull request merged**